src/pages/bug-bounty.tsx

```diff
@@ -0,0 +1,896 @@
+  />
+)
+
+type BountyHuntersArg = { score?: number }
+
+type Node = {
+  readonly name: string
+  readonly username: string
+  readonly score: number
+}
+
+type Client = {
+  title: string
+  link: string
+  image: ImageProps["src"]
+}
+
+type Spec = {
+  title: ReactNode
+  link: string
+}
+
+const sortBountyHuntersFn = (a: BountyHuntersArg, b: BountyHuntersArg) => {
+  if (!a.score || !b.score) return 0
+  return b.score - a.score
+}
+
+export const getStaticProps: GetStaticProps<{}, {}> = async (context) => {
+  const { locale } = context
+
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPath("bug-bounty")
+
+  return {
+    props: await serverSideTranslations(locale!, requiredNamespaces),
+  }
+}
+
+const BugBountiesPage = () => {
+  const { pathname } = useRouter()
+  const { t } = useTranslation("page-bug-bounty")
+
+  const consensusBountyHunters: Node[] = consensusData.sort(sortBountyHuntersFn)
+  const executionBountyHunters: Node[] = executionData.sort(sortBountyHuntersFn)
+
+  const bountyHuntersArrayToObject: Record<string, Node> = [
+    ...consensusData,
+    ...executionData,
+  ].reduce((acc, next) => {
+    const name = next.name
+    if (!name) return acc
+
+    if (acc[name]) {
+      return {
+        ...acc,
+        [name]: {
+          ...next,
+          score: acc[name].score + next.score,
+        },
+      }
+    }
+
+    return {
+      ...acc,
+      [name]: next,
+    }
+  }, {})
+
+  // total all counts using name as identifier, then sort
+  const allBounterHunters = Object.values(bountyHuntersArrayToObject).sort(
+    (a, b) => b.score - a.score
+  )
+
+  const clients: Client[] = [
+    {
+      title: "Besu",
+      link: "https://besu.hyperledger.org/en/stable/",
+      image: besu,
+    },
+    {
+      title: "Erigon",
+      link: "https://github.com/ledgerwatch/erigon",
+      image: erigon,
+    },
+    {
+      title: "Geth",
+      link: "https://geth.ethereum.org/",
+      image: geth,
+    },
+    {
+      title: "Lighthouse",
+      link: "https://lighthouse-book.sigmaprime.io/",
+      image: useColorModeValue(lighthouseLight, lighthouseDark),
+    },
+    {
+      title: "Lodestar",
+      link: "https://chainsafe.github.io/lodestar/",
+      image: lodestar,
+    },
+    {
+      title: "Nimbus",
+      link: "https://our.status.im/tag/nimbus/",
+      image: nimbus,
+    },
+    {
+      title: "Nethermind",
+      link: "https://docs.nethermind.io/nethermind/",
+      image: nethermind,
+    },
+    {
+      title: "Prysm",
+      link: "https://prylabs.net/",
+      image: prysm,
+    },
+    {
+      title: "Teku",
+      link: "https://pegasys.tech/teku",
+      image: useColorModeValue(tekuDark, tekuLight),
+    },
+  ].map((client) => {
+    client.image.width = 60
+    return client
+  })
+
+  const clientsSmall = clients.map((client) => {
+    const thumbnail = client.image as StaticImageData
+    thumbnail.width = 24
```

Comment:
I'm seeing a minimum width of 60px in prod

Prod:
<img width="945" alt="Screenshot 2023-11-28 at 12 57 07" src="https://github.com/ethereum/ethereum-org-fork/assets/62268199/86d41211-277c-4ee2-ab2d-01f8a178fe1a">

This PR:
<img width="810" alt="Screenshot 2023-11-28 at 12 57 11" src="https://github.com/ethereum/ethereum-org-fork/assets/62268199/2c7c2682-e630-47dd-ba5e-4c010758c988">

Reply:
Patched!